### ISSUE TITLE
[en][talent][hunter] trueshot aura - too big increase in attack power

### ISSUE THREAD

**Issue by blinkmaster** [Atlantiss/BugTracker#9526]
**Description**: http://www.wowhead.com/spell=193526/trueshot
gives more than 20% attack power boost

**Current behaviour**: gives too much

**Expected behaviour**: should give 20% increase

**Steps to reproduce the problem**:

1.  get stats from hunter with no trueshot aura - attack power -14144
2.  use trueshot aura - attack power - 17 768
3.  calculate: 14.144 x 1.2 =  `16973
16973 =/= 17 768

1673 = 120%
17 768 = x

x= 1,274

thus trueshot aura gives 27,4% app buff.

blessing of might works fine, no info on other ap increasing buffs

v high prio.

**Revision**: 5498

**Comment by Psychobird7** [comment]
The spell ID is incorrect. 
Correct spell ID is either 19506 or 31519. 

As for the rest... I know Blessing of Might works fine, however I have no info about other %AP increases.

**Comment by Psychobird7** [comment]
Update. Did some testing with a guildie (hunter).      

We both got the correct AP values with and without both Trueshot and Might.     
My unbuffed AP in Prot spec was 8191, Trueshot/Might brought me to the same value of 9829. 20% increase here should've made it 9829.2, which is consistent. 
After respeccing to Ret, my unbuffed AP was 14409, which Trueshot/Might brought to 17291, which is consistent with the 17290.8 the 20% increase should've made it. 

TLDR: Either parts of your report are erroneous, or this is not a bug.

**Comment by tomatosz** [comment]
Blinkmaster, how did you test this, since you can't really 'turn off' Trueshot Aura? (it sounds like you tested it on the hunter that gives the aura). If you changed to another spec to turn off Trueshot Aura, could the difference be caused by some other talent like One With Nature, or even by (not) applying Aspect of the Hawk?

**Comment by xolon** [comment]
I tested it on my hunter by replacing Trueshot Aura talent with sic' em rank 2.
[Without Trueshot Aura](https://i.imgur.com/hxGWJbr.jpg)
Ranged - 12928 | 1292.8 is 10% |
Melee - 629 | 125.8 is 20% |
[With Trueshot Aura](https://i.imgur.com/NIYe9T5.jpg)
Ranged - | 14220.8 = 12928 + 1292.8 |
Melee - |  754.8 = 629 + 125.8 |
So based on this I can say that it works blizzlike.

**Comment by blinkmaster** [comment]
elan bugit char for sure then! 

closing for now, need to do more testing
